core-sdk-samples/higgs-shop-sample-app/src/constants/index.ts

```diff
@@ -7,10 +7,27 @@ export enum ORDER_PHASES {
     COMPLETE = 'complete',
 }
 
+export enum MODAL_MODES {
+    CLOSED = 'closed',
+    ENTRY = 'entry',
+    UPDATE = 'update',
+    CONFIRM = 'confirm',
+    ENV = 'env',
```

Comment:
This is a bit of a nit.  I know what most of these mean after playing around with the app.  `env` is a bit confusing to me though.  I know it means a user who is using the .env file, but perhaps this can be more descriptive, like `localHosted` or something.

Reply:
I agree that it's vague but `localHosted` sounds more confusing to me. Would the opposite of that be `serverHosted` for the hosted version?

Do you think `localDev` more clear?

Reply:
I think `localDev` is good.  I'm not good at variables names on a whim :)